src/Site/views/shared/container/xforge.navbarApp.js

```diff
@@ -10,49 +10,34 @@ angular.module('xforge.navbarApp', ['bellows.services', 'ui.bootstrap', 'pascalp
     };
   }])
   */
-  .controller('navbarController', ['$scope', 'modalService', '$location', 'sessionService',
-  function ($scope, modalService, $location, ss) {
+  .controller('navbarController', ['$scope', 'modalService', '$location', 'asyncSession', '$window',
+  function ($scope, modalService, $location, ss, $window) {
 
     $scope.helpFilePath = '';
 
     // this function should be run whenever the location changes
     function isHelpFilePresentOnServer() {
-      // couldn't figure out an easy way to get just the pathname from $location,
-      // so using window.location instead
-      var appName = window.location.pathname.split('/')[2];
-      var hashPath = $location.path();
-      var helpFilePathsAvailable = [];
-      if (angular.isDefined(ss.session.helps)) {
-        helpFilePathsAvailable = ss.session.helps.filePaths;
-      }
-
+      var appName = $window.location.pathname.split('/')[2];
```

Comment:
Searching globally for `location.pathname.split` this is the only instance I find (it's possible it's implemented differently elsewhere). Chris intends to delete the navbarApp, but otherwise it would make sense to move this to some service that provides metadata.